### PR TITLE
fix(ticket-deserialization): handle JSON deserialization errors for Ticket.class

### PR DESCRIPTION
**Pull request by tomrist9**
- Added missing 	icketClass field in Ticket class to match JSON structure.
- Implemented no-argument constructor in Ticket class to enable Jackson deserialization.
- Configured Jackson to ignore unknown JSON fields using @JsonIgnoreProperties annotation.
- Updated loadTicketsFromResources method to support JSON arrays and single objects.

These changes ensure robust handling of JSON deserialization and prevent errors from unrecognized fields or missing constructors.

### CODE REVIEW

File: andersen-homework/src/main/java/Main.java
Comment (Darya-Drutsko) [inline andersen-homework/src/main/java/Main.java]: save file name in separate constant(also could be stored in config file)

File: andersen-homework/src/main/java/org/model/Ticket.java
Comment (Darya-Drutsko) [inline andersen-homework/src/main/java/org/model/Ticket.java]: why public modifier here?

File: andersen-homework/src/main/java/org/util/BusTicketValidator.java
Comment (Darya-Drutsko) [inline andersen-homework/src/main/java/org/util/BusTicketValidator.java]: Seems like DAY", "WEEK", "MONTH could be an enum :)

File: andersen-homework/src/main/java/org/util/BusTicketValidator.java
Comment (Darya-Drutsko) [inline andersen-homework/src/main/java/org/util/BusTicketValidator.java]: Also I see that "start date" duplicated many times here, let's create constant for such values

**Review by Darya-Drutsko — COMMENTED** [review]
Good job!

File: andersen-homework/src/main/java/org/model/Ticket.java
Comment (tomrist9) [reply andersen-homework/src/main/java/org/model/Ticket.java]: when i selected LocalDate as a modifier , I get this error in validate method Cannot resolve method 'isEmpty' in 'LocalDate' therefore I used string as a modifier.

File: andersen-homework/src/main/java/org/util/BusTicketValidator.java
Comment (tomrist9) [reply andersen-homework/src/main/java/org/util/BusTicketValidator.java]: done

File: andersen-homework/src/main/java/org/model/Ticket.java
Comment (Darya-Drutsko) [reply andersen-homework/src/main/java/org/model/Ticket.java]: I mean why have you chosen here 'public' access modifier, not 'private'?

File: andersen-homework/src/main/java/org/util/BusTicketValidator.java
Comment (Darya-Drutsko) [reply andersen-homework/src/main/java/org/util/BusTicketValidator.java]: Add please constants for duplicated values such as "start date", "ticket type", "price"

File: andersen-homework/src/main/java/org/model/Ticket.java
Comment (tomrist9) [reply andersen-homework/src/main/java/org/model/Ticket.java]: i did not do it intentionally, so I ll change it to private

Comment (tomrist9) [reply andersen-homework/src/main/java/org/model/Ticket.java]: because when i chose private, i get this error in validate method 'ticketType' has private access in 'org.model.Ticket'

File: andersen-homework/src/main/java/org/util/BusTicketValidator.java
Comment (tomrist9) [reply andersen-homework/src/main/java/org/util/BusTicketValidator.java]: have created constant

File: andersen-homework/src/main/java/org/model/Ticket.java
Comment (tomrist9) [reply andersen-homework/src/main/java/org/model/Ticket.java]: now i have solved it with get method

File: andersen-homework/src/main/java/Main.java
Comment (tomrist9) [reply andersen-homework/src/main/java/Main.java]: I have added ticket filename as a configurable property in application.properties

Comment (Darya-Drutsko) [reply andersen-homework/src/main/java/Main.java]: Don't see the result in the pr

File: andersen-homework/src/main/java/org/model/Ticket.java
Comment (Darya-Drutsko) [reply andersen-homework/src/main/java/org/model/Ticket.java]: Don't see the result in the pr

Comment (tomrist9) [reply andersen-homework/src/main/java/org/model/Ticket.java]: check it again please, teacher

File: andersen-homework/src/main/java/Main.java
Comment (tomrist9) [reply andersen-homework/src/main/java/Main.java]: check it again teacher please

**Review by Darya-Drutsko — APPROVED** [review]
Approved, but remove please .idea folder from git and return back files, related to previous homeworks :)
After this changes merge the pull request without additional review from my side

**Pull request merged**
